string baxter_location
Action[] plan
float64 start_time
---
Plan[] plans